GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -76,6 +88,7 @@ GDTCORNetworkType GDTCORNetworkTypeMessage() {
     }
   }
 #endif
+  // Leaves unknown for network connection type on watchOS
```

Comment:
We probably don't want to reserve this network type to watchOS, so I think this comment should be removed.

Reply:
Removed.